app/helpers/application_helper.rb

```diff
@@ -3,6 +3,18 @@ def page_text_direction
     I18n.t("i18n.direction", locale: I18n.locale, default: "ltr")
   end
 
+  def t_locale_fallback(key, options = {})
+    options['locale'] = I18n.locale
+    options[:fallback] = false
+    translation = I18n.t(key, options)
+
+    if translation.nil? || translation.include?("translation missing")
+      I18n.default_locale
+    else
+      false
```

Comment:
Can we use `nil` instead of `false` please? I found that `false` was cast to a string and displayed.
```suggestion
      nil
```

Reply:
> Can we use `nil` instead of `false` please? I found that `false` was cast to a string and displayed.

Fixed.